### DISCUSSION TITLE
Is NWB the right place for cell imaging within labs running organoid experimentation?

### DISCUSSION THREAD

**Discussion by davidparks21** [NeurodataWithoutBorders/helpdesk#52] [General]
I'm from the Braingeneers lab in UC Santa Cruz. We work with Organoids which produce fairly large scale electrophysiology data. We're organizing data formats across 5 universities working with us. I'm recommending NWB as the format published data format for data in the forms: ephys, calcium imaging, patch clamp, and EEG. These data types seem quite in line with the NWB data format, so I'm comfortable making that recommendation (we have someone doing a POC for these data types in our labs right now).

My question is whether our cell imaging projects should or could also employ NWB, and whether you think it's the right format for imaging. The data consists of many high-resolution jpegs, often taken at a variety of focal lengths (called a z-stack). 

My initial impression is that this isn't far from calcium imaging, and after all NWB is just H5 and supports extensions, so I'm sure the answer can be "yes", but I'm trying to figure out if it *should be* yes. In particular does NWB currently house data like this? Perhaps general cell histology to go with the various ephys data?

I'd love to hear from people in this forum.

**Comment by bendichter** [comment]
@davidparks21 thanks for reaching out. You are correct that time series data like ephys, calcium imaging, patch clamp, and EEG fit quite well into NWB. NWB specializes in time series data, so it may not be the optimal format for images of cells that are not measuring activity over time. One could store such data in NWB, and if you want to we would be happy to work with you to do so, but I would also recommend you look into other formats, such as OME, and in particular the new cloud-optimized [OME-Zarr format](https://www.biorxiv.org/content/10.1101/2023.02.17.528834v1).
